content/src/debug.js

```diff
@@ -80,7 +91,7 @@ var debug = window.ide = {
 
     if (name === "seeeval") { reportSeeeval.apply(null, data); }
 
-    if (name === "enter") { stuckComplexity.calls += 1; }
+    if (name === "enter") { reportEnter.apply(null, data); stuckComplexity.calls += 1; }
```

Comment:
Now that there is a reportEnter function, move `stuckComplexity.calls += 1` into it.